src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutes.scala

```diff
@@ -101,13 +101,6 @@ trait UserRoutes extends UserInfoDirectives {
                 }
               } ~
               pathPrefix("petServiceAccount") {
-                pathEndOrSingleSlash {
```

Comment:
nothing should be using this. it has been deprecated and hidden from swagger for months